triton/apps/llms.rst

```diff
@@ -0,0 +1,220 @@
+LLMs
+====
+
+
+.. highlight:: bash
+
+Large-language models are AI models that can understand and generate text
+using transformer architectures.
+
+Because the model weights are typically very large and the interest in the
+models is high, we provide our users pre-downloaded model weights and
+instructions on how to run inference and training on the models.
+
+
+Pre-downloaded model weights
+----------------------------
+Raw model weights
+~~~~~~~~~~~~~~~~~
+We have downloaded the following models weights (PyTorch model checkpoint directories):
+
+.. list-table::
+  :header-rows: 1
+  :widths: 1 1 3 2
+
+  * * Model type
+    * Model version
+    * Module command to load
+    * Description
+
+  * * Llama 2
+    * Raw data
+    * ``module load model-llama2/raw-data``
+    * Raw weights of `Llama 2 <https://ai.meta.com/llama/>`__.
+
+  * * Llama 2
+    * 7b
+    * ``module load model-llama2/7b``
+    * Raw weights of 7B parameter version of `Llama 2 <https://ai.meta.com/llama/>`__.
+
+  * * Llama 2
+    * 7b-chat
+    * ``module load model-llama2/7b-chat``
+    * Raw weights of 7B parameter chat optimized version of `Llama 2 <https://ai.meta.com/llama/>`__.
+
+  * * Llama 2
+    * 13b
+    * ``module load model-llama2/13b``
+    * Raw weights of 13B parameter version of `Llama 2 <https://ai.meta.com/llama/>`__.
+
+  * * Llama 2
+    * 13b-chat
+    * ``module load model-llama2/13b-chat``
+    * Raw weights of 13B parameter chat optimized version of `Llama 2 <https://ai.meta.com/llama/>`__.
+
+  * * Llama 2
+    * 70b
+    * ``module load model-llama2/70b``
+    * Raw weights of 70B parameter version of `Llama 2 <https://ai.meta.com/llama/>`__.
+
+  * * Llama 2
+    * 70b-chat
+    * ``module load model-llama2/70b-chat``
+    * Raw weights of 70B parameter chat optimized version of `Llama 2 <https://ai.meta.com/llama/>`__.
+
+Each module will set the following environment variables:
+
+- ``MODEL_ROOT`` - Folder where model weights are stored, i.e., PyTorch model checkpoint directory.
+- ``TOKENIZER_PATH`` - File path to the tokenizer.model. 
+
+Here is an example slurm script using the raw weights to do batch inference. For detailed environment setting up, example prompts and python code, please check out `this repo <https://github.com/AaltoSciComp/llm-examples/tree/main/batch-inference-llama2>`__.
```

Comment:
My suggestion is to also link to the [slurm documentations](https://scicomp.aalto.fi/triton/tut/slurm/). It can be useful for people who are new to this concept.